drivers/sensor/lsm6dso/Kconfig

```diff
@@ -85,4 +85,7 @@ config LSM6DSO_EXT_LPS22HB
 
 endif # LSM6DSO_SENSORHUB
 
+config LSM6DSO_ENABLE_FIFO
```

Comment:
I would put the FIFO parameters in the DT. This for the usual reason that this driver is multi-instance and one instance may be configured with FIFO on and another with FIFO off.